ajax/libs/echarts-gl/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "echarts-gl",
+  "description": "Extension pack of ECharts providing 3D plots and globe visualization",
+  "version": "1.1.2",
+  "maintainers": [
+    {
+      "name": "Yi Shen"
+    }
+  ],
```

Comment:
We don't support the `maintainers` attr, would it work to use the `author` attr, which we do support?